packages/web/src/components/GameBody.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState, useRef, useEffect } from "react";
+import Boot from "game/scenes/boot";
+import GameScene from "game/scenes/game";
+import Preload from "game/scenes/preload";
+import { IonPhaser } from '@ion-phaser/react';
+import DesktopWindow from "components/DesktopWindow";
+import Phaser from "phaser";
+
+
+export default function GameBody(props: {gameConfig?: Phaser.Types.Core.GameConfig}) {
+    const gameRef = useRef(null)
```

Comment:
I think you might be able to do this to avoid the type casts

```suggestion
    const gameRef = useRef<Phaser.Game>(null)
```